### PR TITLE
Add description and short description to search api

### PR DESCRIPTION
**Pull request by jonhartzler**
Previously both fields were left nil.

### CODE REVIEW

**Comment by jgondron** [comment]
:+1:

File: spec/models/waggle/adapters/solr/search/hit_spec.rb [hunk 1]
```diff
@@ -36,4 +36,48 @@
...
+      expect(subject.description).to be_kind_of(String)
+    end
+
+    it "feteches the description" do
+      expect(subject).to receive(:fetch_text).with(:description)
+      subject.description
+    end
+  end
+
+  describe "description" do
```
Comment (rfox2) [inline spec/models/waggle/adapters/solr/search/hit_spec.rb]: Did you mean to repeat this example?

File: spec/models/waggle/search/hit_spec.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      end
+
+      it "returns the data from the adapter_hit" do
+        expect(subject.send(field)).to eq(adapter_hit.send(field))
+      end
+    end
+  end
+
+  describe "#short_description" do
+    it "calls the appropriate methods on the adapter_hit" do
```
Comment (rfox2) [inline spec/models/waggle/search/hit_spec.rb]: Not sure I understand the purpose of this test

**Comment by rfox2** [comment]
:+1:

**Pull request merged**